### DISCUSSION TITLE
微软登陆失败“你未在Minecraft官网创建档案，或没有购买Minecraft”

### DISCUSSION THREAD

**Discussion by duoduo13123678** [Meloong-Git/PCL#3444] [提问] [answered]
显示微软登陆失败“你未在Minecraft官网创建档案，或没有购买Minecraft”是为什么？

**Comment by burningtnt** [comment, marked as answer]
字面意思。你没有在 minecraft.net 初始化过自己的账户，或根本没有购买 Java 版本。

**Comment by MoYuan-CN** [comment]
这代表你所登录的 Microsoft 账号没有购买 Minecraft
&nbsp;&nbsp;&nbsp;&nbsp;\- 如果是这样，请确认你的 Minecraft 购买时所使用的 Microsoft 账号，登录那个
或者确认所登陆的 Microsoft 账号有购买 Minecraft
&nbsp;&nbsp;&nbsp;&nbsp;\- 请到 [minecraft.net](https://www.minecraft.net/zh-hans) 登录账号后创建 Java 版 Minecraft 档案（填写名字，上传皮肤等）

**Comment by duoduo13123678** [comment]
我已经购买minecraft了，那为什么呢
![屏幕截图 2024-03-13 201815](https://github.com/Hex-Dragon/PCL2/assets/143173417/36244c2d-6a04-4b46-956f-42521d084054)

**Reply by WorldHim** [reply]
同样经常遇到这种情况
个人经验
尝试打开官网并登录成功后再通过 PCL 启动

**Comment by allMagicNB** [comment]
用官方启动器登录账号再用 PCL2 试试
